includes/namespace.php

```diff
@@ -159,6 +164,18 @@ function remove_site( $error, $site ) {
  * @return void
  */
 function deactivate( $network_wide ) {
+	// Remove capabilities while the post type is still registered.
+	$injector = Services::get_injector();
+	if ( method_exists( $injector, 'make' ) ) {
+		$capabilities = $injector->make( User\Capabilities::class );
+		$capabilities->remove_caps_from_roles();
+	}
```

Comment:
This data is removed on uninstall. We should keep it on deactivate.

Reply:
Argh, I didn't see it in uninstall.php at first.